src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -624,30 +575,31 @@ public sealed partial class CharEnumerator : System.Collections.Generic.IEnumera
         public bool MoveNext() { throw null; }
         public void Reset() { }
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(32767), Inherited = true, AllowMultiple = false)]
+    [System.AttributeUsageAttribute((System.AttributeTargets)(32767), Inherited=true, AllowMultiple=false)]
     public sealed partial class CLSCompliantAttribute : System.Attribute
     {
         public CLSCompliantAttribute(bool isCompliant) { }
         public bool IsCompliant { get { throw null; } }
     }
     public delegate int Comparison<in T>(T x, T y);
     public delegate TOutput Converter<in TInput, out TOutput>(TInput input);
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential, Size=1)]
     public readonly partial struct DateTime : System.IComparable, System.IComparable<System.DateTime>, System.IConvertible, System.IEquatable<System.DateTime>, System.IFormattable, System.Runtime.Serialization.ISerializable
```

Comment:
where is the `LayoutKind.Sequential` coming from here? DateTime normally has `LayoutKind.Auto`.

Reply:
Before this change the ref didn't include a StructLayout on this type and when I round-tripped the ref through genapi it picked up these default values from the old ref. It is probably worth going through and fixing these up based on the implementation.

Reply:
The StructLayout attribute is not really part of the contract today.

Reply:
You think it is better to just filter it out then?

Reply:
I think so